src/Main.js

```diff
@@ -0,0 +1,177 @@
+
+class Main {
```

Comment:
you didn't implement a separate render method, that would simplify this file a lot.
this way you can have only one or two dedicated functions that manipulates the dom and renders the whole list, so after reorder, removing one, removing all or adding 4 new tasks you would call the same function - renderList that can handle it all by simply rerender the whole list (removing and building it again)

Reply:
I understand, but it isn't inefficient? each adding/removing/reordering to remove and render **all** of the tasks?